internal/core/renderer.rs

```diff
@@ -25,13 +25,16 @@ impl<T: RendererSealed> Renderer for T {}
 /// users to re-implement these functions.
 pub trait RendererSealed {
     /// Returns the size of the given text in logical pixels.
-    /// When set, `max_width` means that one need to wrap the text so it does not go further than that
+    /// When set, `max_width` means that one need to wrap the text, so it does not go further than that.
+    /// When set, `wrap_anywhere` means that the text wrapping will occur at any given character, instead of
+    /// only at word boundaries.
     fn text_size(
         &self,
         font_request: crate::graphics::FontRequest,
         text: &str,
         max_width: Option<LogicalLength>,
         scale_factor: ScaleFactor,
+        wrap_anywhere: bool,
```

Comment:
My preference here would be to pass the `TextWrap` as enum, instead of just a boolean.

Reply:
I agree, my only thought is regarding the max_width, if it should still be a Option or not. 

Prior to this, max_width was being used to define if using no-wrap or word-wrap, but if we are passing the TextWrap enum, it would be redundant to use a option. 

Beyound that, passing a word-wrap with max_width = none should fail, but currently the method doesn't return a result.

What do you think is the best way to approach this?

I'm inclined to change:
wrap_anywhere: bool -> text_wrap: TextWrap
max_width: Option<LogicalLength> -> max_width: LogicalLength

and with this, when using no-wrap, setting LogicalLength to zero.

Reply:
My feeling is the latter is clearer (the one without option). It's internal API though:)